pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -186,6 +187,8 @@ func (c *Controller) Run(workers int, stopCh <-chan struct{}) {
 		return
 	}
 
+	endpointslicemetrics.RegisterMetrics()
```

Comment:
Is it common practice to do it in Run?

I believe this can be done in init()?

Reply:
I'm not 100% sure if this works in init(), I was just going off of the few examples I found in other controllers (https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/persistentvolume/pv_controller_base.go#L302 and https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/attachdetach/attach_detach_controller.go#L363)

Reply:
Moved it to init, I think it does make more sense there.